src/styles/blocks/stars.css

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background: url("../../images/star.svg") center no-repeat;
+}
+
+.stars__star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--4 :not(:last-child) {
+  background: var(--star-active);
+}
```

Comment:
You set stars--4 at html, however you need to be flexible and if you get a card with 3 stars, you will just set stars--3 and you've already have styles for this case.
So, just move star styles from previous task here

Reply:
I did this in a commit [a0014d6](https://github.com/mate-academy/layout_product-cards/pull/2914/commits/a0014d6e71117103d6f32e818fd849e88171ca90), but the mentor told me to do one block